cmd/build/build.go

```diff
@@ -60,6 +60,11 @@ var BuildCmd = &cobra.Command{
 
 		err = bgit.Add("bsf/")
 		if err != nil {
+			if err == bgit.ErrFilesNotAddedToVersionControl{
+				fmt.Println(styles.ErrorStyle.Render(err.Error() + "\n Please ensure all necessary files are added/committed in your version control system"))
```

Comment:
Does this mean if I don't "git add" all the files in my working directory, we'll error out? 🤔  
If so, we don't want that..

Reply:
never mind, I got confused due to the error message